packages/core/src/filters/FilterSystem.ts

```diff
@@ -389,7 +389,7 @@ export class FilterSystem extends System
      * @param {number} [resolution] override resolution of the renderTexture
      * @returns {PIXI.RenderTexture}
      */
-    getFilterTexture(input: RenderTexture, resolution: number): RenderTexture
+    getFilterTexture(input?: RenderTexture, resolution?: number): RenderTexture
```

Comment:
ok, agree. The point is to force `input` as first param, but its not required. Some old filters pass `resolution` as first.